src/CommunityToolkit.Maui/CommunityToolkit.Maui.csproj

```diff
@@ -75,6 +75,7 @@
 
   <ItemGroup Condition="$(TargetFramework.Contains('-windows'))">
     <PackageReference Include="CommunityToolkit.WinUI.Notifications" Version="7.1.2" />
+    <PackageReference Include="Microsoft.WindowsAppSDK" Version="1.0.0" />
```

Comment:
Should this go in [`CommunityToolkit.Maui.Core.csproj`](https://github.com/CommunityToolkit/Maui/blob/main/src/CommunityToolkit.Maui.Core/CommunityToolkit.Maui.Core.csproj)?

Reply:
good question... AFAIK we need this reference if we need to access the platform controls. At the end of the day I think that is fine to add it on `Core` just in case

Reply:
Right on! Let's remove it from here and move it to `Core`, then. 

Since `CommunityToolkit.Maui` depends on `CommunityToolkit.Maui.Core`, it'll inherit the `Microsoft.WindowsAppSDK` NuGet package.

Reply:
Done